nmostesting/GenericTest.py

```diff
@@ -247,6 +247,16 @@ def check_content_type(self, headers):
                               .format(ctype)
         return True, ""
 
+    def check_accept(self, headers):
+        """Check the Accept header of an API request"""
+        if "Accept" in headers:
+            accept = headers["Accept"]
+            accept_params = accept.split(";")
```

Comment:
I don't think this is quite right, because Accept is a list header.
Need to split list on `,` and then each element on `;`, and account for optional whitespace.
See https://tools.ietf.org/html/rfc7231#section-5.3.2.